.travis.yml

```diff
@@ -4,7 +4,7 @@ dist: trusty
 addons:
   apt:
     sources:
-      - chef-stable-trusty
+      - chef-current-trusty
```

Comment:
I agree for nightly builds if we had them.

Only mentioning it because recently I've had to deal with people installing a stable version of ChefDK locally and then CI borks because it uses a different version of foodcritic or cookstyle.  Asking them to use appbundle-updater or install a current version locally is to match the CI job is unrealistic.

Not going to let this hold things up but think it needs more discussion in next meeting.